packages/client-generator-ts/src/utils/addPreamble.ts

```diff
@@ -5,15 +5,18 @@ const generatedCodePreamble = `
 /* eslint-disable */
 `
 
+const tsNoCheckPreamble = `/* @ts-nocheck */`
```

Comment:
This appears to miss a leading and/or trailing newline.

Reply:
Fixed